test/unit/data/fake_names_index_template.json

```diff
@@ -0,0 +1,395 @@
+{
+  "order": 0,
+  "template": "names",
+  "settings": {},
+  "mappings": {
+    "_default_": {
+      "dynamic_templates": [
+        {
+          "string_fields": {
+            "mapping": {
+              "type": "text",
+              "fields": {
+                "raw": {
+                  "ignore_above": 256,
+                  "type": "keyword"
+                }
+              }
+            },
+            "match_mapping_type": "string",
+            "match": "*"
+          }
+        }
+      ],
+      "properties": {
+        "Company": {
+          "type": "text",
+          "fields": {
+            "raw": {
+              "ignore_above": 256,
+              "type": "keyword"
+            }
+          }
+        },
+        "MoneyGramMTCN": {
+          "index": true,
+          "type": "double",
+          "doc_values": true
+        },
+        "CountryFull": {
```

Comment:
Ah that's right, ES creates text and keywords out of strings by default now doesn't it?